### PR TITLE
Update table to include SNAP

### LINKED ISSUE #1656: Update table to include SNAP
Default sync mode should be SNAP  for everything except `dev`. 

Related to [this PR](https://github.com/hyperledger/besu-docs/pull/1647)

### PR DESCRIPTION
**Pull request by joaniefromtheblock**
## Description

-  Replace FAST sync with SNAP

### Issue(s) fixed

Fixes #1656 

### Preview

-

### CODE REVIEW

File: docs/public-networks/reference/cli/options.md [hunk 1]
```diff
@@ -2640,13 +2640,13 @@ Possible values include the following:
...
+| `sepolia` | ETH   | Test        | [SNAP](#sync-mode) | A PoS network            | Multi-client testnet [Sepolia](https://sepolia.dev)                                  |
+| `lukso`   | ETH   | Production  | [SNAP](#sync-mode) | A PoS network            | Network for the [Lukso chain](https://lukso.network/)                                |
 | `dev`     | ETH   | Development | [FULL](#sync-mode) | A PoW network            | Development network with low difficulty to enable local CPU mining                   |
-| `classic` | ETC   | Production  | [FAST](#sync-mode) | A PoS network            | The main [Ethereum Classic network](https://ethereumclassic.org)                     |
-| `mordor ` | ETC   | Test        | [FAST](#sync-mode) | A PoW network            | Testnet for [Ethereum Classic](https://github.com/eth-classic/mordor)                |
+| `classic` | ETC   | Production  | [SNAP](#sync-mode) | A PoS network            | The main [Ethereum Classic network](https://ethereumclassic.org)                     |
```
Comment (macfarla) [inline docs/public-networks/reference/cli/options.md:2648]:
```suggestion
| `classic` | ETC   | Production  | [SNAP](#sync-mode) | A PoW network            | The main [Ethereum Classic network](https://ethereumclassic.org)                     |
```

**Review by macfarla — APPROVED** [review]
LGTM except ETC (classic) is PoW

**Comment by joaniefromtheblock** [comment]
Thanks @macfarla

**Review by bgravenorst — APPROVED** [review]
lgtm

**Pull request merged**